app/models/group.rb

```diff
@@ -0,0 +1,6 @@
+class Group < ApplicationRecord
+  belongs_to :user
+  has_and_belongs_to_many :expenses
+
```

Comment:
- [ ] The database schema should reflect the following structure:

![image](https://user-images.githubusercontent.com/79058364/227439921-57baa020-2dfb-42d7-9613-d4a21a80577a.png)

- [ ] According to the Entity relational database Diagram, there is a many to many relationship between Group and Entity so, You should have created a join table to link the two tables. Kindly address this 💯 😉 
- Note: In Rails and PostgreSQL, creating a join table is necessary for modeling many-to-many relationships because it allows us to establish a connection between two tables without duplicating data.

Reply:
```rb
class CreateJoinTableGroupExpenses < ActiveRecord::Migration[7.0]
  def change
    create_join_table :groups, :expenses do |t|
      t.index [:group_id, :expense_id]
      t.index [:expense_id, :group_id]

      t.timestamps
    end
  end
end
```